hyrule/control.hy

```diff
@@ -1,10 +1,54 @@
 (require
   hyrule.macrotools [defmacro/g! defmacro!])
 (import
-  hyrule.collections [prewalk coll?]
+  hyrule.anaphoric [recur-sym-replace]
+  hyrule.collections [prewalk by2s]
+  hyrule.iterables [coll?]
   hyrule.misc [inc])
 
 
+(defmacro branch [tester #* rest]
```

Comment:
I wanted to name it `clauses`, but I thought it would be too confusing because I would expect "clause" to mean a case-result pair, and the cases and results are coming into the function unpaired in a single sequence (look, ma, no brackets!). Was I overthinking it?